pymongo/client_session.py

```diff
@@ -461,6 +471,10 @@ def callback(session, custom_arg, custom_kwarg=None):
         however, ``with_transaction`` will return without taking further
         action.
 
+        This :class:`ClientSession` is **not thread-safe or fork-safe**. The
+        ``callback`` must not attempt to execute multiple operations
+        concurrently.
```

Comment:
Suggest rephrasing to something like:

:class:`ClientSession` instances are **not thread-safe or fork-safe**.  Consequently, the ``callback`` must not attempt to use the session for executing multiple concurrent operations

Reply:
Done.